cinema/serializers.py

```diff
@@ -1,6 +1,6 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Genre, Actor, CinemaHall
 
 
 class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer`, `GenreSerializer`, `ActorSerializer`, and `CinemaHallSerializer`. This will automatically generate the `create` and `update` methods and reduce boilerplate code.